docs/website/docs/getting-started/tflite.md

```diff
@@ -15,8 +15,117 @@ python -m pip install \
   iree-tools-tflite
 ```
 
-- [Command Line](./tflite-cmd.md)
-- [Python API](./tflite-python.md)
+## Command line
+IREE's tooling is divided into two components: import and compilation.
+
+1. The import tool converts the TFLite flatbuffer to an IREE compatible form,
+validating that only IREE compatible operations remain. Containing a combination of TOSA
+and IREE operations.
+2. The compilation stage generates the bytecode module for a list of targets, which can
+be executed by IREE.
+
+These two stages can be completed entirely via the command line.
```

Comment:
The overview here could be pulled up a level, as it applies to both the command line and python API. You could also then nest the CLI and Python API sections one level below that (heading level 2 `##` -> heading level 3 `###`) 

```
## Prerequisites
<pip install>

## Importing and Compiling
<intro paragraph / numbered list>

### Using Command Line Tools
<CLI section>

### Using the Python API
<Python API section>
```